TAO/tests/GIOP_Fragments/Big_String_Sequence/echo.mpc

```diff
@@ -0,0 +1,34 @@
+// -*- MPC -*-
+project(*idl): taoidldefaults {
+  IDL_Files {
+    Echo.idl
+  }
+  custom_only = 1
+}
+
+project(*Server): taoserver, namingexe, iortable, utils, avoids_corba_e_micro {
```

Comment:
base projects need cleanup, way too much looking briefly at the test code

Reply:
The only base project I was able to remove without causing the build to fail was `avoids_corba_e_micro`. Note,
this whole example is just a copy of `examples/Simple/echo` with some tweaks to reproduce this bug. We don't
use MPC in our own projects and I don't know enough about MPC to do more than copy existing stuff with small
and obvious adjustments. I've pushed a change to remove  `avoids_corba_e_micro`.